lib/ohai/plugins/windows/system_enclosure.rb

```diff
@@ -18,13 +18,11 @@
 
 Ohai.plugin :SystemEnclosure do
   provides "system_enclosure"
+  depends "dmi"
 
   collect_data(:windows) do
-    require "wmi-lite/wmi"
     system_enclosure Mash.new
-    wmi = WmiLite::Wmi.new
-    wmi_object = wmi.first_of("Win32_SystemEnclosure").wmi_ole_object
-    system_enclosure[:manufacturer] = wmi_object.invoke("manufacturer")
-    system_enclosure[:serialnumber] = wmi_object.invoke("serialnumber")
+    system_enclosure[:manufacturer] = get_attribute(:dmi, :chassis, :manufacturer)
```

Comment:
We should cut an issue in cookstyle to get people using node['system_enclosure']['manufacturer'] onto the DMI equiv in the future. That way we can eventually remove this plugin namespace